### DISCUSSION TITLE
how to set setCredential = True in $fetch

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17211] [Questions]
$fetch(`${api.value}/api/auth/csrf/`)
how can i set seCredential in $fetch method?

**Comment by amrnn90** [comment]
You can pass the `credentials` option:
https://developer.mozilla.org/en-US/docs/Web/API/fetch#credentials

```ts
$fetch(url, {
    credentials: "include",
});
```
